web-animations-2/Overview.bs

```diff
@@ -3028,21 +3049,51 @@ with which it is associated is updated.
 
 </ins>
 
-<h3 id="the-animatable-interface">The <code>Animatable</code> interface</h3>
+The <code>Animatable</code> interface mixin {#the-animatable-interface-mixin}
+----------------------------------------
 
-<div class="methods">
+The {{KeyframeAnimationOptions}} dictionary interface
+is modified to add the following members:
 
-:   <dfn method for=Animatable lt="getAnimations()">
-    sequence&lt;Animation&gt; getAnimations()</dfn>
-::  Add:
+<pre class='idl'>
+dictionary TimelineRangeOffset {
+  CSSOMString? rangeName;
+  CSSNumericValue offset;
+};
 
-    > If this object is the [=effect target/target element=] of two or more
-    > <a>animation effects</a> which are associated with the
-    > same <a>animation</a>, the corresponding {{Animation}}
-    > object will still only appear in the returned list once.
+partial dictionary KeyframeAnimationOptions : KeyframeEffectOptions {
+    (TimelineRangeOffset or CSSNumericValue or DOMString) rangeStart = "normal";
+    (TimelineRangeOffset or CSSNumericValue or DOMString) rangeEnd = "normal";
+};
+</pre>
 
+<div class="members">
+:   <dfn dict-member for=KeyframeAnimationOptions>rangeStart</dfn>
+::  An optional value which, if present,
+    specifies the start of the <a>animation</a>’s [=timeline attachment range=].
```

Comment:
[=timeline attachment range=] doesn't link when I check this out locally. Is it just a stale index or does it need to specify this comes from scroll-animations?

Reply:
It'll eventually link up: WA is going to need to set up the conceptual framework for attachment ranges (which Scroll Animations and CSS Animations 2 will link to), and that will define this term. But I wanted to get these bits of the API edits in first, since they're more concrete.

Reply:
No problem, sounds good.